exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -291,6 +300,46 @@
             var url = protocol + "//" + host + requestPath;
             return url;
       }
+
+      //Iterates through all the nodes for update
+      function reloadMemory () {
+          for (i = 1; i <= size; i++) {
+              var address = $("#row-"+i).find("#address").contents().get(0).nodeValue.trim();
+              updateMemory(address, i);
+          }
+      }
+
+      //Update memory
+      function updateMemory(drillbit,idx) {
+        var result = $.ajax({
+          type: 'GET',
+          url: "http://"+drillbit+":8047/status/metrics",
```

Comment:
You can not hard-code port, it can be changed using property. See how graceful shutdown is checking drillbits statuses.

Reply:
Good point. I'll follow the same port extraction logic.

Reply:
Actually, this is a challenge. We don't seem to be exposing any information on the http-port.  The only information I am getting back into DrillbitInfo are the other 3 ports: control, data and user. If I need to had HTTP to this (which I think it should), we need to modify the protobuf (UserBitShared). :(
Do you have any suggestions?

Reply:
So one way to find out a Drillbit's HTTP port is through this call:
`http://localhost:8047/portNum `
to get 
```
{
  "port" : 8047
}
```
Notice how I need to use the port number in the request's URL to get that very number! 🤣